content/ai/how-i-hooked-ai-video-generation-into-my-dev-workflow-with-higgsfield/_index.md

```
@@ -0,0 +1,293 @@
+
+A server name, `type: http`, and a URL. That's the entire connector. The first time anything calls a Higgsfield tool, Claude Code prompts for an OAuth login. One time, then you forget about it. Same shape, same endpoint plugs into [Claude Desktop](https://claude.com), [OpenCode](https://opencode.ai), or whatever MCP-aware agent you happen to use. Different agents, more or less the same result.
+
+The connector is six lines and you're done. The skill is where the actual workflow lives. Let's take a look.
+
+```sh
+cat .claude/skills/manuscript-broll-suggest/SKILL.md
+```
+
+The output is as follows (trimmed for brevity).
+
+~~~md
```

markdownlint-cli2 (0.22.1)

[warning] Code fence style
Expected: backtick; Actual: tilde

(MD048, code-fence-style)